hw/dv/sv/cip_lib/doc/index.md

```diff
@@ -425,8 +425,11 @@ class ip_env_cfg extends cip_base_env_cfg #(.RAL_T(ip_reg_block));
 ```
 
 ### Security Verification for REGWEN CSRs
-If the REGWEN CSR is HW read-only, it can be fully verified by the common [csr_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/csr_tests.hjson).
-If it’s a HW updated CSR, users need to write a test to verify it separately since cip_lib and dv_base_reg can’t predict its value.
+If the REGWEN CSR meets the following criteria, it can be fully verified by the common [csr_tests](https://github.com/lowRISC/opentitan/blob/master/hw/dv/tools/dvsim/tests/csr_tests.hjson).
+ - The REGWEN CSR and its related lockable CSRs are HW read-only registers.
+ - The related lockable CSRs are not WO type, otherwise, read value is always 0 and CSR tests can't really verify if the write value is taken or not.
+ - No CSR exclusion is added to the REGWEN CSR and its related lockable CSRs.
```

Comment:
Maybe:
```suggestion
 - No CSR exclusion has been added to the REGWEN CSR and its related lockable CSRs.
```
(or alternatively "No CSR exclusion applies to..."?)

Reply:
To make it more precise - "No CSR exclusions have been..." since there could be more than one.

Reply:
Updated, thanks